crates/node/src/cli.rs

```diff
@@ -37,52 +47,45 @@ pub struct ExtendedRunCmd {
     pub run_cmd: RunCmd,
 
     /// Load a custom chain-spec from an url
-    #[clap(long)]
-    pub chain_spec_url: Option<String>,
-
-    /// Load a custom index.json file for downloading custom assets
-    /// Check documentation in
-    /// https://github.com/keep-starknet-strange/madara/blob/main/docs/configs.md
-    #[clap(long)]
-    pub configs_url: Option<String>,
-
-    /// Disable madara default configs:
-    /// - Fetching index.json, genesis.json and genesis assets
-    /// - Fetching default chain specs
-    #[clap(long)]
-    pub disable_madara_configs: bool,
-
-    /// Disable automatic url fetching for madara config files
-    #[clap(long)]
-    pub disable_url_fetch: bool,
+    /// If you want to load a chain spec that is present in your filesystem, use `--chain=<PATH>`
+    #[clap(long, conflicts_with = "testnet")]
+    pub fetch_chain_spec: Option<String>,
 
     /// Path to the folder where all configuration files and data are stored
-    /// There is an alias for madara_path <-> base_path
-    /// Default: ~/.madara
-    #[clap(long)]
+    /// base_path will always be overwritten by madara_path
+    /// in the case you use the --tmp, the base_path will be changed during the runtime
```

Comment:
the `--tmp` flag

Reply:
`he base_path will be changed during the runtime` to what?

Reply:
non deterministic